Build/Chakra.Build.Default.props

```diff
@@ -3,10 +3,15 @@
   <Import Project="Common.Build.Default.props"/>
   <Import Condition="'$(Clang)'!=''" Project="Chakra.Build.Clang.Default.props"/>
   <PropertyGroup>
-    <WindowsTargetPlatformVersion Condition="'$(Platform)'=='ARM'">10.0.10240.0</WindowsTargetPlatformVersion>
+    <!-- Default the installed latest Win10 SDK -->
+    <WindowsTargetPlatformVersion_10>$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Microsoft SDKs\Windows\v10.0@ProductVersion)</WindowsTargetPlatformVersion_10>
+    <!-- Sometimes the version in the registry has to .0 suffix, and sometimes it doesn't. Check and add it -->
+    <WindowsTargetPlatformVersion_10 Condition="'$(WindowsTargetPlatformVersion_10)' != '' and !$(WindowsTargetPlatformVersion_10.EndsWith('.0'))">$(WindowsTargetPlatformVersion_10).0</WindowsTargetPlatformVersion_10>
+    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)'==''">$(WindowsTargetPlatformVersion_10)</WindowsTargetPlatformVersion>
 
-    <!-- Always use Platform SDK for core builds -->
-    <EventManifestXmlPath>$(WindowsSDK80Path)Include\um</EventManifestXmlPath>    
+    <!-- Unfortunately, ARM build depends on ksarm.h, which is only available in 10.0.10240.0 -->
```

Comment:
@kfarnung Thanks for tips.  I modified the installation and got it working now.
I have changed the commit to use the latest install SDK if the file ksarm.h exists.  If not, default back to 10.0.10240.0.

Also I have made it so that we will only set the target version if it hasn't set yet.